rest/handler.go

```diff
@@ -688,6 +695,11 @@ func checkJWTIssuerStillValid(ctx context.Context, dbCtx *db.DatabaseContext, us
 // checkAdminAuthenticationOnly simply checks whether a username / password combination is authenticated pulling the
 // credentials from the handler
 func (h *handler) checkAdminAuthenticationOnly() (bool, error) {
+	if h.privs == adminPrivs && !*h.server.Config.API.AdminInterfaceAuthentication {
```

Comment:
Is this change required?  This looks like it's already being evaluated by the called (if shouldCheckAdminAuth).

Reply:
nope